src/main/resources/mixins.clientcommands.json

```diff
@@ -65,7 +65,9 @@
     "commands.alias.ClientSuggestionProviderMixin",
     "commands.enchant.MultiPlayerGameModeMixin",
     "commands.findblock.ClientLevelMixin",
+    "commands.generic.ClientPacketListenerMixin",
     "commands.generic.CommandSuggestionsMixin",
+    "commands.reply.ChatScreenMixin",
```

Comment:
I don't think `ClientPacketListenerMixin` belongs in `commands.generic`. It can just go in `commands.reply`.